Make forms error message adhere to styleguide

huangy22 opened the pull request:
For Refs #3167

Edit the form error messages to adhere to the styleguide:
- Removes instances of "Please"
- Removes . on short single sentences

pradyunsg commented on tests/unit/test_forms.py:
```diff
@@ -69,12 +69,12 @@ def test_good_passwords(self, password):
             (
                 "bombo!b",
-                ("Password is too easily guessed. Add another word or two. "
+                ("Password is too easily guessed. Add another word or two."
```
This doesn't seem right, there should be a space between the two sentences?

huangy22 replied:
You are right. I made a mistake here. Sorry about that.

di commented on warehouse/accounts/forms.py:
```diff
@@ -145,13 +145,13 @@ def validate_email(self, field):
         domain = field.data.split('@')[-1]
         if domain in disposable_email_domains.blacklist:
             raise wtforms.validators.ValidationError(
                 "Sorry, you cannot create an account with an email address "
```
I think we could probably lose the "Sorry" here as well.

di commented on warehouse/accounts/forms.py:
```diff
@@ -84,13 +84,13 @@ def validate_password(self, field):
                 raise wtforms.validators.ValidationError(
                     "There have been too many unsuccessful login attempts, "
-                    "please try again later."
+                    "Try again later."
```
We either need to put a period at the end of the previous phrase, or lowercase this `Try`.

huangy22 commented:
@di Thank you so much for the comments! I have been a little busy attending a data science conference recently, sorry for my reply. I have addressed your comments, let me know if there's anything else to change!

di commented on tests/unit/accounts/test_forms.py:
```diff
@@ -237,8 +237,8 @@ def test_blacklisted_email_error(self):
-            "Sorry, you cannot create an account with an email address from "
-            "this domain. Please use a different email."
+            "Sorry, you can't create an account with an email address from "
+            "this domain. Use a different email."
```
This should be:
```diff
-            "Sorry, you cannot create an account with an email address from "
-            "this domain. Please use a different email."
+            "You cannot create an account with an email address from "
+            "this domain. Use a different email."
```

di commented:
@huangy22 I'm still seeing a lot of changes in the following files that are unnecessary:

* All the changes in `tests/unit/test_forms.py`
* All the changes in `tests/unit/test_views.py`

huangy22 commented:
@di Thank you so much for your comments and accepting the PR! I have one question though, somehow I seem to not be able to find myself on the contributor page and I wonder if there's any steps I'm missing here?

di commented:
@huangy22 If you're talking about https://github.com/pypa/warehouse/graphs/contributors, it seems like that page only shows at most 100 contributors ordered by number of commits (even though we have 104).

Since each PR gets squashed, this one only counts as a single commit. My guess is since your contribution was relatively recent, you're one of the 4 contributors that don't show up on that page.
